service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/NoticeOfProceedingsSubmittedControllerTest.java

```diff
@@ -0,0 +1,65 @@
+package uk.gov.hmcts.reform.fpl.controllers;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.autoconfigure.OverrideAutoConfiguration;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.reform.fpl.service.FeatureToggleService;
+import uk.gov.service.notify.NotificationClient;
+import uk.gov.service.notify.NotificationClientException;
+
+import static org.mockito.ArgumentMatchers.anyMap;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static uk.gov.hmcts.reform.fpl.NotifyTemplates.NOTICE_OF_PROCEEDINGS_ISSUED_JUDGE_TEMPLATE;
+import static uk.gov.hmcts.reform.fpl.utils.CoreCaseDataStoreLoader.callbackRequest;
+
+@ActiveProfiles("integration-test")
+@WebMvcTest(NoticeOfProceedingsController.class)
+@OverrideAutoConfiguration(enabled = true)
+class NoticeOfProceedingsSubmittedControllerTest extends AbstractControllerTest {
+
+    private static final String ALLOCATED_JUDGE_EMAIL_ADDRESS = "judge@gmail.com";
```

Comment:
minor: Don't really need this to be static anymore since it's only used the once.

Reply:
this broke checkstyle when i didn't keep as static so leaving as is